frontend/pages/editOrganization/[organizationUrl].tsx

```diff
@@ -57,6 +64,25 @@ export default function EditOrganizationPage({
   const locationInputRef = useRef(null);
   const [locationOptionsOpen, setLocationOptionsOpen] = useState(false);
 
+  // add all sectors that are assigned to the organization to the possible sectors
+  // so that, when editing a project with e.g. specific sectors all sectors - even
+  // hub specific ones are available
+  if (organization && organization.sectors) {
+    for (const sector_mapping of organization.sectors) {
+      if (!sector_mapping || !sector_mapping.sector) {
+        continue;
+      }
+      const sector = sector_mapping.sector as SectorOptionType;
+      // match by sector.key
+      const exists = allSectors.find((s) => s.key === sector.key);
+      if (!exists) {
+        allSectors.push(sector);
+      }
+    }
+    // sort sectors by name
+    allSectors.sort((a, b) => (a.name < b.name ? -1 : 1));
+  }
+
```

Comment:
Moving this directly into getInitiatialprops. Altering a prop that is passed from getInitialProps in here doesn't seem like the right place to me. Especially since this code could be re-run on re-render